tests/pools/test_pool_cmdline.py

```diff
@@ -0,0 +1,36 @@
+# flake8: noqa: E501
```

Comment:
Note that this file does not test all plotnft subcommands, because most of them require txch in a wallet, and on-chain actions.